src/Athena.Importer/data/offering/ecnm152-p100-f2018.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "c22cd6a4-b7f2-437e-bf45-09e774ae97cd",
```

Comment:
An offering with the same ID exists for `@{id=8588ca76-eaa0-43d0-bead-ad1c3374a881; name=MTHM  280 - LINEAR ALGEBRA;}`

Reply:
ok